CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- added: Avascan as evmScan server for Avalanche
```

Comment:
`changed: Avascan replaces snowtrace as an evmScan server and explorer for Avalanche`